packages/nft-playbook/src/info.json

```diff
@@ -1,7 +1,4 @@
 {
```

Comment:
Would suggest stronger naming of file like Blockchain_config or something like that